integrations/standalone/tests/pageobjects/ResponsibleSelect.ts

```diff
@@ -0,0 +1,57 @@
+import { Locator, Page, expect } from '@playwright/test';
+
+export class ResponsibleSelect {
+  private readonly page: Page;
+  private readonly parentLocator: Locator;
+  private readonly label: string;
+  private readonly type: Locator;
+  private readonly script: Locator;
+  private readonly combo: Locator;
+
+  constructor(page: Page, parentLocator: Locator, label: string) {
+    this.page = page;
+    this.parentLocator = parentLocator;
+    this.label = label;
+    this.type = parentLocator.getByLabel(this.label).first();
+    this.script = this.locator().getByRole('textbox');
+    this.combo = this.locator().getByRole('combobox').nth(1);
+  }
+
+  async fill(value: string) {
+    await this.script.fill(value);
+  }
+
+  async clear() {
+    await this.chooseType('Role');
+    await this.combo.click();
+    await this.locator().getByRole('option', { name: 'Everybody' }).first().click();
+  }
+
+  async chooseType(type: 'Role' | 'Role from Attr.' | 'User from Attr.' | 'Nobody & delete') {

```

Comment:
you can reuse this type from the protocol package (a little bit of typescript magic):
```ts
type ObjectValuesAsUnion<T> = {
  [V in keyof T]: T[V];
}[keyof T];

async chooseType(type: ObjectValuesAsUnion<typeof RESPONSIBLE_TYPE>) {
```

The first one is a typehelper, maybe we should move this to the editor package or something like that...